bindings/rust/extended/s2n-tls-sys/templates/Cargo.template

```diff
@@ -36,8 +36,8 @@ stacktrace = []
 # unstable-foo = []
 
 [dependencies]
-aws-lc-rs = { version = "1" }
-libc = "0.2"
+aws-lc-rs = { version = "1.6.4" }
```

Comment:
aws-lc-rs 1.6.4 is needed because the [DEP_AWS_LC environment variables were not exported before then](https://github.com/aws/aws-lc-rs/pull/335), and [they're needed in order to build s2n-tls-sys](https://github.com/aws/s2n-tls/blob/1c38961ddca45243b73c684dd4ddcc9ae454b2e3/bindings/rust/extended/s2n-tls-sys/build.rs#L198-L200).